iron-image.html

```diff
@@ -77,6 +79,12 @@
       position: relative;
     }
 
+    #img {
+      display: block;
+      width: var(--iron-image-width, auto);
```

Comment:
we could include width attribute if available on the host, and if not defined, use the custom css prop:
```css
width: var(var(attr(width), --iron-image-width), auto);